### PR TITLE
Added WebView2 115 items to "Overview of APIs"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article sections for review:

* Added new h4 section **Navigation kind** under h2 section "Navigate to pages and manage loaded content" section: 
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733#navigation-kind

* Instead of adding `service worker` enum value, added `BrowsingDataKinds` enum in a new mini-section "Clearing browsing data" within the **User data** section/tab:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733#user-data

The new items are from RelNotes 115 > Release Stable:
https://learn.microsoft.com/microsoft-edge/webview2/release-notes#101901177

Outline of article (find "NEW SECTION" below):
```
## Main classes: Environment, Controller, and Core

## Web/native interop
#### Host/web object sharing
#### Script execution
#### Web messaging
#### Script dialogs
#### Shared buffer

## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Permissions
#### Context menus
#### Status bar
#### User Agent
#### Autofill
#### Audio
#### Swipe gesture navigation
#### Fullscreen
#### PDF toolbar
#### Theming
#### Language
#### New window
#### Close window
#### Document title
#### Favicon 
#### Security and privacy
###### Tracking prevention
###### SmartScreen
###### Custom crash reporting

## Process management

## Navigate to pages and manage loaded content
#### Manage content loaded into WebView2
#### Navigation history
#### Navigation kind              <-- NEW SECTION
#### Block unwanted navigating
#### Navigation events
#### Manage network requests in WebView2
#### Custom scheme registration
#### Client certificates
#### Server certificates
#### Launch an external URI scheme

## iframes

## Authentication

## Rendering WebView2 in non-framework apps
#### Sizing, positioning, and visibility
#### Zooming
#### Rasterization scale
#### Focus and tabbing
#### Parent window
#### Keyboard accelerators
#### Default background color

## Rendering WebView2 using Composition
#### Connecting to the visual tree
#### Forwarding input
#### Drag and drop
#### Accessibility

## User data   <-- NEW mini-SECTION within tab to add b.d.k. enum
#### Multiple profiles

## Performance and debugging
#### Memory usage target

## Chrome DevTools Protocol (CDP)
```

AB#45737697

### CODE REVIEW

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 1]
```diff
@@ -1347,6 +1347,32 @@ The history methods allow back and forward navigation in WebView2, and the histo
 ---
 
 
+<!-- ------------------------------ -->
+#### Navigation kind
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1351]: Is the new Navigation kind section in right spot in outline?
Rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733&tabs=dotnetcsharp#navigation-kind

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 2]
```diff
@@ -2260,6 +2286,51 @@ Access and manipulate the profile:
 ---
 
 
+<!-- ------------------------------ -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:2289]: Omit or include the "service workers" enum value (section Clearing service worker browsing data) for use by ClearBrowsingData?  Delete this new section?  Move the 1 link into an above tab-set?
Rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733&tabs=dotnetcsharp#clearing-service-worker-browsing-data

Outline is at top of main PR page.

@novac42 @peiche-jessica @aluhrs13

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 1 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:1351]: This being in the "Navigate to pages and manage loaded content" LGTM.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 2 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:2289]: This is an addition to the existing CoreWebView2BrowsingDataKinds. I would not explicitly call this enum value out unless we do for the rest of the enum values. No strong opinion if we want to keep this as is.

**Review by peiche-jessica — APPROVED** [review]

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:2289]: The existing article doesn't mention "datakinds", so this section, which focuses on 1 member of that enum, is the only place that this enum is represented, and this new spot might overemphasize this one new member.  

Maybe the enum overall should be better represented in the article, w/ less focus on this particular "serviceworkers" member.

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: The result is good, b/c it creates a mini-section "Clearing browsing data" within the existing section/tab:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733&tabs=dotnetcsharp#user-data
image

**Pull request merged**